src/StaticValidator.php

```diff
@@ -52,6 +52,12 @@ public static function getPluginManager()
      */
```

Comment:
Latest commit:

- documents `@throws`
- Renames `$args` to `$options`
- Better documents `$options`